test/utils.dart

```diff
@@ -7,7 +7,7 @@ import 'package:yaml/src/equality.dart' as equality;
 import 'package:yaml/yaml.dart';
 
 /// A matcher that validates that a closure or Future throws a [YamlException].
-final Matcher throwsYamlException = throwsA(new isInstanceOf<YamlException>());
+final Matcher throwsYamlException = throwsA(TypeMatcher<YamlException>());
```

Comment:
I believe there are still frontends in use, at least within Google, that don't yet support optional `new`.

Also, if you are using it, you should update the SDK constraint to exclude any SDKs where frontends don't use Dart 2 by default.

Reply:
Adding `new` back